babel.config.json

```diff
@@ -0,0 +1,115 @@
+{
+  "babelrcRoots": [
+    ".",
+    "./packages/*"
+  ],
```

Comment:
With this, do we still need the rootMode option? Reading the Babel docs it seems to indicate that it would search from the roots upwards.

Reply:
ya.. we still need the `rootMode` otherwise babel wont look up the tree for the `babel.config.js` file.

Here are the docs for configuring babel in a mono-repo
https://babeljs.io/docs/en/config-files#monorepos